addon-test-support/audit-if.ts

```diff
@@ -13,6 +14,15 @@ export default function a11yAuditIf(
   contextSelector?: ElementContext | RunOptions | undefined,
   axeOptions?: RunOptions | undefined
 ): PromiseLike<void> {
+  deprecate(
+    "ember-a11y-testing's `a11yAuditIf` is deprecated. Please use a11yAudit when using the `enableA11yAudit` query param",
+    false,
+    {
+      id: 'ember-a11y-testing-deprecated-a11y-audit-if',
+      until: '5.0.0',
+    }
+  );
```

Comment:
Maybe we could tell folks to themselves use `forceA11yAudit() && await a11yAudit();` themselves? I mean, its kinda silly, but it should work...

Reply:
That doesn't sound like a user friendly API. I'd prefer to not do this.

Reply:
We spoke offline and have agreed that this addon will provide the primitives for users to build and invoke their own `a11yAuditIf`-like functionality.